bookshelf/tests/CloudStorageTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Samples\Bookshelf;
+require_once __DIR__.'/../vendor/autoload.php';
```

Comment:
Rather than include `autoload` in each test file, we should create a `test/bootstrap.php` file, and specify in `phpunit.xml` to include this file before running each test ([example](https://github.com/bshaffer/oauth2-server-php/blob/develop/phpunit.xml#L12))